addons/mail/models/mail_followers.py

```diff
@@ -100,7 +100,7 @@ def _get_recipient_data(self, records, message_type, subtype_id, pids=None):
         self.env['res.users'].flush(['notification_type', 'active', 'partner_id', 'groups_id'])
         self.env['res.partner'].flush(['active', 'partner_share'])
         self.env['res.groups'].flush(['users'])
-        self.env['mail.channel'].flush(['email_send', 'channel_type'])
+        self.env['mail.channel'].flush(['channel_type'])
```

Comment:
Why do we keep this flush ? Seems we could get rid of it now. Does not have much impact or remaining flow and simplifies a bit code.